--- Mac/Makefile	2011-03-31 23:07:50.000000000 +0200
+++ Mac/Makefile	2011-03-31 23:06:31.000000000 +0200
@@ -182,15 +182,15 @@
 	cd IDLE && make install
 
 install_BuildApplet:
+	#$(RUNSHARED)  $(BUILDPYTHON) $(srcdir)/scripts/BuildApplet.py \
+	#	--destroot "$(DESTDIR)" \
+	#	--python=$(prefix)/Resources/Python.app/Contents/MacOS/Python \
+	#	--output "$(DESTDIR)$(PYTHONAPPSDIR)/Build Applet.app" \
+	#	$(srcdir)/scripts/BuildApplet.py
+#ifneq ($(LIPO_32BIT_FLAGS),)
+#	rm "$(DESTDIR)$(PYTHONAPPSDIR)/Build Applet.app/Contents/MacOS/Python"
+#	lipo $(LIPO_32BIT_FLAGS) -output "$(DESTDIR)$(PYTHONAPPSDIR)/Build Applet.app/Contents/MacOS/Python" $(BUILDPYTHON)
+#endif   
-	$(RUNSHARED)  $(BUILDPYTHON) $(srcdir)/scripts/BuildApplet.py \
-		--destroot "$(DESTDIR)" \
-		--python=$(prefix)/Resources/Python.app/Contents/MacOS/Python \
-		--output "$(DESTDIR)$(PYTHONAPPSDIR)/Build Applet.app" \
-		$(srcdir)/scripts/BuildApplet.py
-ifneq ($(LIPO_32BIT_FLAGS),)
-	rm "$(DESTDIR)$(PYTHONAPPSDIR)/Build Applet.app/Contents/MacOS/Python"
-	lipo $(LIPO_32BIT_FLAGS) -output "$(DESTDIR)$(PYTHONAPPSDIR)/Build Applet.app/Contents/MacOS/Python" $(BUILDPYTHON)
-endif   
 
 MACLIBDEST=$(LIBDEST)/plat-mac
 MACTOOLSDEST=$(prefix)/Mac/Tools
