--- projects.pro	2010-11-06 02:55:11.000000000 +0100
+++ projects_removed.pro	2011-02-17 08:00:15.452466468 +0100
@@ -9,9 +9,9 @@
 
 isEmpty(QT_BUILD_PARTS) { #defaults
     symbian {
-       QT_BUILD_PARTS = libs tools examples demos
+       QT_BUILD_PARTS = libs tools
     } else {
-       QT_BUILD_PARTS = libs tools examples demos docs translations
+       QT_BUILD_PARTS = libs tools docs translations
     }
 } else { #make sure the order makes sense
    contains(QT_BUILD_PARTS, translations) {
@@ -36,10 +36,6 @@
 for(PROJECT, $$list($$lower($$unique(QT_BUILD_PARTS)))) {
     isEqual(PROJECT, tools) {
        SUBDIRS += tools
-    } else:isEqual(PROJECT, examples) {
-       SUBDIRS += examples
-    } else:isEqual(PROJECT, demos) {
-       SUBDIRS += demos
     } else:isEqual(PROJECT, libs) {
        include(src/src.pro)
     } else:isEqual(PROJECT, docs) {
